tests/foreman/cli/test_puppetmodule.py

```diff
@@ -1,117 +0,0 @@
-"""Test class for PuppetModule CLI
-
-:Requirement: Puppetmodule
-
-:CaseAutomation: Automated
-
-:CaseLevel: Acceptance
-
-:CaseComponent: Puppet
-
-:Assignee: vsedmik
-
-:TestType: Functional
-
-:CaseImportance: High
-
-:Upstream: No
-"""
-import pytest
-
-from robottelo.cli.factory import make_product
-from robottelo.cli.factory import make_repository
-from robottelo.cli.puppetmodule import PuppetModule
```

Comment:
Do we want to remove this class? But again, if you search for it, it appears at a number of places, all need to be reviewed

Reply:
I think I already answered this above.